train.py

```diff
@@ -9,19 +9,56 @@
 
 import collections
 import math
+import sys
 import random
+from datetime import datetime
 
 import numpy as np
 import torch
-
-from fairseq import checkpoint_utils, distributed_utils, options, progress_bar, tasks, utils
+import torch_xla
+import torch_xla.debug.metrics as met
+import torch_xla.distributed.data_parallel as dp
```

Comment:
Can we remove unused imports like `data_parallel`?

Reply:
yep, will do, thanks.